src/qibolab/instruments/zhinst.py

```diff
@@ -758,15 +761,14 @@ def measure_relax(self, exp, qubits, relaxation_time, acquisition_type):
                         pulse.zhpulse.uid += str(i)
 
                         """Integration weights definition or load from the chip folder"""
-                        weights_file = Path(
-                            str(qibolab_folder)
-                            + f"/runcards/{self.chip}/weights/integration_weights_optimization_qubit_{q}.npy"
```

Comment:
@Jacfomg I don't know if here you need a specific folder to save the weights. For now I only fix tests.

Reply:
I will need a place to store them, maybe the repository or database they are working on ?

Reply:
Indeed, it would be good to move this to the new repository/new database and also find a way to use it in other drivers, as it should be relevant of all instruments.

Reply:
Until we move to the database we could have a fixed folder.
Currently I am using what we use to store instruments data.